mod_pywebsocket/standalone.py

```diff
@@ -361,6 +361,16 @@ def _build_option_parser():
                         type=int,
                         default=_DEFAULT_REQUEST_QUEUE_SIZE,
                         help='request queue size')
+    parser.add_argument(
+        '--handler-encoding',
+        '--handler_encoding',
+        dest='handler_encoding',
+        type=six.text_type,
+        default=None,
```

Comment:
What do you think about defaulting to utf-8?

Reply:
I didn't want to break backwards-compatibility in case someone was depending on the current behaviour.

Unfortunately I broke backwards compatibility for Python 2.7 a bit by accident, as `io.open()` rejects invalid characters, whereas `open()` in version 2.7 doesn't.

Reply:
I see, thank you.